colcon_output/event_handler/console_cohesion.py

```diff
@@ -36,26 +35,26 @@ def __init__(self):  # noqa: D107
         satisfies_version(
             EventHandlerExtensionPoint.EXTENSION_POINT_VERSION, '^1.0')
         self.enabled = ConsoleCohesionEventHandler.ENABLED_BY_DEFAULT
+        self._lines = defaultdict(list)
 
     def __call__(self, event):  # noqa: D102
         data = event[0]
 
-        # instead of buffering all output in memory
-        # read the combined log file written by the `log` event handler
-        if isinstance(data, JobEnded):
+        if isinstance(data, StdoutLine):
```

Comment:
Any reason not to do:
```suggestion
        if isinstance(data, (StdoutLine, StderrLine)):
```

Reply:
No reason at all. I accepted the proposed change and in 76323b228d528bcd220ff37827aa558ece065621 removed the now obsolete `StderrLine` block.